docs/autonomy.md

```diff
@@ -23,66 +23,106 @@ Commands:
-Usage: autonomy deploy build deployment [OPTIONS] PUBLIC_ID KEYS_FILE
+Usage: autonomy deploy build deployment [OPTIONS] PUBLIC_ID_OR_HASH KEYS_FILE
 
   Build deployment setup for n agents.
 
 Options:
-  --o PATH            Path to output dir.
-  --n INTEGER         Number of agents.
-  --docker            Use docker as a backend.
-  --kubernetes        Use docker as a kubernetes.
-  --packages-dir PATH  Path to packages folder (For local usage).
-  --dev               Create development environment.
-  --force             Remove existing build and overwrite with new one.
-  --help              Show this message and exit.
+  --o PATH             Path to output dir.
+  --n INTEGER          Number of agents.
+  --docker             Use docker as a backend.
+  --kubernetes         Use kubernetes as a backend.
+  --packages-dir PATH  Path to packages folder (for local usage).
+  --dev                Create development environment.
+  --version TEXT       Specify deployment version.
+  --force              Remove existing build and overwrite with new one.
+  --skip-images        Specify whether to build images or not.
+  --remote             To use a remote registry.
+  --local              To use a local registry.
+  -p                   Ask for password interactively
+  --password PASSWORD  Set password for key encryption/decryption
+  --help               Show this message and exit.
+```
+
+To create an environment you'll need a service id and a file containing keys with funds for the chain you want to use. The necessary images will be built during the build process.
+
+`PUBLIC_ID_OR_HASH` refers to the public id of the service, eg. `valory/oracle_hardhat`. `KEYS_FILE` refers the the file containing funded keys for the agent instances.
+
+Example keys file
+
+```json
+[
+    {
+        "address": "0xf39Fd6e51aad88F6F4ce6aB8827279cffFb92266",
+        "private_key": "0xac0974bec39a17e36ba4a6b4d238ff944bacb478cbed5efcae784d7bf4f2ff80"
+    },
+    {
+        "address": "0x70997970C51812dc3A010C7d01b50e0d17dc79C8",
+        "private_key": "0x59c6995e998f97a5a0044966f0945389dc9e86dae88c7a8412f4603b6b78690d"
+    },
+    {
+        "address": "0x3C44CdDdB6a900fa2b585dd299e03d12FA4293BC",
+        "private_key": "0x5de4111afa1a4b94908f83103eb1f1706367c2e68ca870fc3fb9a804cdab365a"
+    },
+    {
+        "address": "0x90F79bf6EB2c4f870365E785982E1f101E93b906",
+        "private_key": "0x7c852118294e51e653712a81e05800f419141751be58f605c371e15141b007a6"
+    }
+]
 ```
 
-To create an environment you'll need a service id and a file containing keys with funds for the chain you want to use.
+*These keys are taken from `deployments/keys/hardhat_json.keys` file and can be used for a local deployment*
 
 ```bash
 # create a docker deployment
-$ autonomy deploy build deployment valory/oracle_hardhat deployments/keys/hardhat_keys.json
+$ autonomy deploy build deployment valory/oracle_hardhat deployments/keys/hardhat_keys.json --packages-dir packages/ --local
```

Comment:
How does the reader get the "packages" folder?

Reply:
So I wanted to use the IPFS registry but this command syntax is not being checked by the IPFS hash check script, So I thought I i'll create a separate PR to fix this. But I think we can update this manually for a while and create a task for david v to update the check script.

Reply:
Yes manual today and then open issue for @dvilelaf